cli/src/pcluster/api/controllers/cluster_operations_controller.py

```diff
@@ -444,3 +445,11 @@ def _create_message(failure_reason, action_needed):
     if action_needed:
         message = f"{message}. {action_needed}" if message else action_needed
     return message or "Error during update"
+
+
+def _get_creation_failures(cluster_status, cfn_stack):
+    """Get a list of Failure objects containing failure code and reason when cluster creation failed."""
+    if cluster_status != ClusterStatus.CREATE_FAILED:
+        return None
+    failure_code, failure_reason = cfn_stack.get_cluster_creation_failure()
```

Comment:
question, from this code it seems we will always have just a failure in the `failures` field, is that correct? Why?

Reply:
```
    if cluster_status != ClusterStatus.CREATE_FAILED:
        return None
```
When cluster creation doesn't failed, it return None, there will not be failures field.

When cluster creation fail, there will always have a failures field